python/ray/dag/compiled_dag_node.py

```diff
@@ -112,7 +115,27 @@ def do_exec_tasks(
             if done:
                 break
             for operation in schedule:
+                start_t = time.time()
                 done = tasks[operation.local_idx].exec_operation(self, operation.type)
+                end_t = time.time()
+
+                if RAY_ADAG_ENABLE_PROFILING:
```

Comment:
`do_exec_tasks` executes remotely, and the env var is obtained from the worker/actor process. If there are multiple nodes, the env var may not be correctly set?

Reply:
it is a very good point. I think we should make this an argument of dag ideally.

Reply:
Oh that's correct, currently we need to add it to `runtime_env` when calling `ray.init` when using multiple nodes. So let's keep this env var for now?